src/components/layout/nav-drawer.tsx

```diff
@@ -0,0 +1,93 @@
+// Modules
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+// Types
+type Item = {
+  name: string;
+  icon: JSX.Element;
```

Comment:
The Item type is being repeatedly redefined maybe separated into a type directory